integ/run-integ.ps1

```diff
@@ -250,7 +250,7 @@ Function Clean-Test {
     )
 
     # Run "docker-compose down" once the tests have been ran.
-    docker-compose --file $DockerComposeFilePath --profile $CoreProfile --profile $TestProfile down -v --rmi all --remove-orphans
```

Comment:
Why skip cleanup?

Reply:
Well it was causing some weird errors during the test run wherein few image layers were cleaned up during first test say for `cloudwatch` and then during second test for `cloudwatch_logs`, cleanup+test fails saying that the layers cannot be found.